docs/_documentations/mdt-vsc-uninstall.md

```diff
@@ -1,34 +1,21 @@
 ---
 layout: docs
 title: Uninstalling Codewind for VS Code
-description: How to uninstall Codewind from VS Code
+description: Uninstalling Codewind for VS Code
 keywords: uninstall, remove, delete, tools, eclipse, uninstalling Codewind for VS Code
 duration: 1 minute
 permalink: mdt-vsc-uninstall
 type: document
-order: 70
-parent: mdt-vsc-overview
 ---
 
-# Uninstalling Codewind from VS Code
-
-Remove the Codewind containers and images
-
-1. Ensure any running containers are stopped.  In the **Codewind view**, bring up the context menu on the **Codewind** item and click **Stop Codewind**
-2. Remove the Docker images
-    - On Windows, enter
-    ```bash
-    %USERPROFILE%\.vscode\extensions\ibm.codewind-0.2.0\bin\installer\windows\codewind-installer.exe remove
-    ```
-    - On macOS or Linux, enter
-    ```bash
-    $HOME/.vscode/extensions/ibm.codewind-0.2.0/bin/installer/macos/codewind-installer remove
-    ```
+# Uninstalling Codewind for VS Code
+Remove the Codewind containers and images:
+1. Stop all running containers. From the **Local** folder in **Codewind** view, click **Stop Local Codewind**.
```

Comment:
It's not a folder. We can call it the "local codewind [tree] item" or something like that.

Reply:
I have changed it to "From the local Codewind tree item..."

Reply:
Change the second instance too